packages/macro/src/macroJs.test.ts

```diff
@@ -262,6 +262,47 @@ describe("js macro", () => {
       })
     })
 
+    it("plural with variable only cases", () => {
+      const macro = createMacro()
+      const exp = parseExpression(
+        "plural(count, { one: `# glass of ${drink}`, other: drink})"
+      )
+      const tokens = macro.tokenizeChoiceComponent(exp as CallExpression)
+      expect(tokens).toEqual({
+        type: "arg",
+        name: "count",
+        value: expect.objectContaining({
+          name: "count",
+          type: "Identifier",
+        }),
+        format: "plural",
+        options: {
+          one: [
+            {
+              type: "text",
+              value: "# glass of ",
+            },
+            {
+              type: "arg",
+              name: "drink",
+              value: expect.objectContaining({
+                name: "drink",
+                type: "Identifier",
+              }),
+            },
+          ],
+          other: {
+            type: "arg",
+            name: "drink",
+            value: expect.objectContaining({
+              name: "drink",
+              type: "Identifier",
+            }),
+          },
+        },
+      })
+    })
+
```

Comment:
You can safely drop unit tests from macroJs.test.ts file. They don't have a lot of sense, it's perfectly covered from fixtures with js code and this just create more work for maintenance.